build/deploy/examples/minimum.jsonnet

```diff
@@ -3,19 +3,20 @@ local dss = import '../../deploy/dss.libsonnet';
 local metadataBase = import '../../deploy/metadata_base.libsonnet';
 
 local metadata = metadataBase {
-  namespace: 'dss-main',
+  namespace: 'namespace',
   clusterName: 'your_cluster_context',
   cockroach+: {
     hostnameSuffix: 'db.your_hostname_suffix.com',
     locality: 'your_unique_locality',
     nodeIPs: ['0.0.0.0', '1.1.1.1', '2.2.2.2'],
     balancedIP: '3.3.3.3',
-    shouldInit: true,
+    shouldInit: true, //Set to false if joining a cluster
+    JoinExisting: ['0.db.westus.example.com', '1.db.westus.example.com', '2.db.westus.example.com' ],
```

Comment:
Let's add a comment noting what the user should replace these with

Reply:
Done.